config/locales/en.yml

```diff
@@ -703,6 +703,9 @@ en:
         - id: '0048'
           text: What to do if you’ve been furloughed
           href: https://www.gov.uk/guidance/coronavirus-covid-19-what-to-do-if-youre-employed-and-cannot-work
+        - id: '0048'
```

Comment:
Having the same id as the previous element look suspicious, but have check with Hanna and through the code, and the ids support a feature that is now redundant - so just a place holder.

However, I'd think it better that a unique number is used, as duplicating the number from the previous element suggests that there is a special relationship between this elements share this id. I'd suggest something obviously different. Like `'1049'`